tests/unit/moduleapi/commandfilter.tcl

```diff
@@ -54,6 +54,17 @@ start_server {tags {"modules"}} {
         r commandfilter.retained
     } {my-retained-string}
 
+    test {Command Filter post callback} {
```

Comment:
this test doesn't test that the filter was executed or that it did operate correctly.
we could add some check to validate the right lines were printed to the log file, but i wonder if we could do better (without using the log file).
i.e. the filter will keep that indication elsewhere for the test to access it with some command.

we could do RM_Call to LPUSH them into some list, and access them with LRANGE, but i suppose it's invalid to call any commands from within the post filter.
the other alternative is to store that in some global and add a command that accesses that.

that brings be to the realization that i'm not sure what this hook is good for (or maybe i forgot)
i.e. what do we expect a module to be able to do from within that callback.
@jjz921024 @guybe7 please comment